README.md

```diff
@@ -18,3 +18,30 @@ CONFIG_BT_NIMBLE_ENABLED=y
 
 - To enable Extended advertising, additionally append `CONFIG_BT_NIMBLE_EXT_ADV=y`.<br>
   (For use with ESP32C3, ESP32S3, ESP32H2 ONLY)
+
+### Configuring for iOS Auto-Reconnect
+
+For auto-reconnection of iOS devices to a BLE server, specific settings in the `sdkconfig` file and the Rust code are required.
+
+#### Update `sdkconfig`
+
+Add the following line to your `sdkconfig` file:
+
+```
+CONFIG_BT_NIMBLE_NVS_PERSIST=y
+```
+
+This ensures the persistence of bonding information in the device's non-volatile storage (NVS), allowing iOS devices to automatically reconnect without needing to rebond after a system reset or power cycle.
+
+#### Set Device Security Options in Rust
+
+In your Rust implementation, set the security options as follows:
+
+```rust
+device
+  .security()
+  .set_auth(AuthReq::Bond) // or .set_auth(AuthReq::Bond | AuthReq:Mitm)
+  .set_io_cap(SecurityIOCap::NoInputNoOutput) // Any option here works
+```
+
+Here, `.set_auth(AuthReq::Bond)` is used to enable bonding, which stores security keys for future reconnections. Note that the use of `.set_auth(AuthReq::Sc)` (Secure Connections) prevents iOS from reconnecting.
```

Comment:
In isssue #24, we concluded that if `sm_our_key_dist` and `sm_their_key_dist` are set, it is possible to reconnect even if Secure Connections is used.
These settings can be set by using the [`resolve_rpa()`](https://taks.github.io/esp32-nimble/esp32_nimble/struct.BLESecurity.html#method.resolve_rpa) function.

If it is possible to reconnect using this function in your environment, please modify this comment.